### PR TITLE
MM-41865 Creating a new explore for Calls Plugin Telemetry

### PR DESCRIPTION
**Pull request by sri-byte**
1) Impact; This is new explore for Calls Plugin Telemetry. Creating this explore from the events schema. From this new explore we will be able to create new dashboards for Calls Plugin Telemetry data.
2) Testing: The data has been loaded into the database manually, once this PR is approved, I will build the dashboard and get it reviewed by the Calls team and create a separate PR for the backend code.

### CODE REVIEW

File: views/mattermost/server_fact.view.lkml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 view: server_fact {
-sql_table_name: mattermost.server_fact ;;
```
Comment (jasonblais) [inline views/mattermost/server_fact.view.lkml:2]: Are changes in this file intended for this PR?

**Review by jasonblais — COMMENTED** [review]
Just one question below, thanks Sri!

File: views/mattermost/server_fact.view.lkml (hunk 1 above)
Comment (sri-byte) [reply views/mattermost/server_fact.view.lkml:2]: You're right @jasonblais, I thought the changes would create a new PR, I removed these changes from this one.

Comment (sri-byte) [reply views/mattermost/server_fact.view.lkml:2]: I'll create a new one for these changes, once this PR gets merged.

Comment (jasonblais) [reply views/mattermost/server_fact.view.lkml:2]: @sri-byte If you create a new git branch and submit your changes for server_fact table there, you can submit a separate PR for this second git branch. That way you don't have to wait until this PR is merged. I can simply approve the one for the server_fact table.

Let me know if you have any questions about using multiple git branches for separate PRs.

File: views/events/calls_events.view.lkml [hunk 2]
```diff
@@ -0,0 +1,217 @@
+# The name of this view in Looker is "Calls Events"
+view: calls_events {
+  # The sql_table_name parameter indicates the underlying database table
+  # to be used for all fields in this view.
+  sql_table_name: "EVENTS"."CALLS_EVENTS"
+    ;;
+  drill_fields: [id]
+  # This primary key is the unique key for this table in the underlying database.
+  # You need to define a primary key in a view in order to join to other views.
```
Comment (catalintomai) [inline views/events/calls_events.view.lkml:9]: these look to be generic comments (are they coming from the template?), should we remove them?

File: views/events/calls_events.view.lkml [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+  dimension: id {
+    primary_key: yes
+    type: string
+    sql: ${TABLE}."ID" ;;
+  }
+
+  # Here's what a typical dimension looks like in LookML.
+  # A dimension is a groupable field that can be used to filter query results.
```
Comment (catalintomai) [inline views/events/calls_events.view.lkml:18]: I would remove the first two rows.

File: views/events/calls_events.view.lkml [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+  }
+
+  dimension: duration {
+    type: number
+    sql: ${TABLE}."DURATION" ;;
+  }
+
+  # A measure is a field that uses a SQL aggregate function. Here are defined sum and average
+  # measures for this dimension, but you can also add measures of many different aggregates.
+  # Click on the type parameter to see all the options in the Quick Help panel on the right.
```
Comment (catalintomai) [inline views/events/calls_events.view.lkml:93]: remove?

File: views/mattermost/server_fact.view.lkml [hunk 5]
```diff
@@ -158,10 +158,10 @@ sql_table_name: mattermost.server_fact ;;
...
     type: yesno
     sql: CASE WHEN datediff(DAY, ${first_active_date}, ${last_active_date}) >= 7 AND ${last_active_date} >= (SELECT MAX(last_active_date - interval '5 day') FROM mattermost.server_fact) THEN TRUE
-  WHEN (datediff(DAY, ${first_active_date}, ${last_active_date}) < 7 AND datediff(DAY, ${first_active_date}, ${last_active_date}) >= 3) AND ${last_active_date} >= (SELECT MAX(last_active_date - INTERVAL '1 DAY') FROM mattermost.server_fact) THEN TRUE
-   WHEN (datediff(DAY, ${first_active_date}, ${last_active_date}) < 3) AND ${last_active_date} = (SELECT MAX(last_active_date) FROM mattermost.server_fact) THEN TRUE
-   WHEN ${paid_license_expire_date} >= CURRENT_DATE THEN TRUE
-    ELSE FALSE END ;;
+        WHEN (datediff(DAY, ${first_active_date}, ${last_active_date}) < 7 AND datediff(DAY, ${first_active_date}, ${last_active_date}) >= 3) AND ${last_active_date} >= (SELECT MAX(last_active_date - INTERVAL '1 DAY') FROM mattermost.server_fact) THEN TRUE
```
Comment (catalintomai) [inline views/mattermost/server_fact.view.lkml:161]: @sri-byte , please fix the spacing to clearly delineate what are the intended changes. Thanks!.

**Review by streamer45 — APPROVED** [review]
Looks good from a data perspective, everything we currently track is here, thanks @sri-byte :+1:

**Review by itao — APPROVED** [review]

File: views/events/calls_events.view.lkml (hunk 3 above)
Comment (catalintomai) [reply views/events/calls_events.view.lkml:18]: I'm mostly referring to comments like "Here's what a typical dimension looks like in LookML.", I assume they are canned comments coming from some template that can be removed and should add instead more useful/topical ones.

**Pull request merged**